VideoWeb/VideoWeb.AcceptanceTests/Pages/PrivateConsultationRoomPage.cs

```diff
@@ -8,7 +8,8 @@ public class PrivateConsultationRoomPage
         public static readonly By LockedBadge = By.CssSelector(".lock-badge.locked");
         public static readonly By IncomingFeed = By.CssSelector("video[id^='incomingFeed']");
         public static readonly By LockButton = By.Id("lockButton");
-        public static readonly By LeavePrivateConsultationButton = By.Id("leaveButton");
+        //public static readonly By LeavePrivateConsultationButton = By.Id("leaveButton");
```

Comment:
I think the commented out line should be removed.

Reply:
Normally I'd agree but I suspect that there may well be a reversal here.  Commented code can be sometimes be useful.